apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -2817,6 +2519,7 @@ mod tests {
 
     #[tokio::test]
     async fn test_handle_error_throttling() {
```

Comment:
Previously this was failing due to the use of the global to hold the dashmap and tests interfering with each other.